tests/transforms/test_experimental/test_transform_program.py

```diff
@@ -115,6 +115,35 @@ def test_iter_program(self):
             assert isinstance(elem, TransformContainer)
             assert elem.transform is first_valid_transform
 
+    def test_add_programs(self):
+        """Test adding two transform programs"""
+        transform_program1 = TransformProgram()
+        transform1 = TransformContainer(transform=first_valid_transform)
+        transform_program1.push_back(transform1)
+
+        transform_program2 = TransformProgram()
+        transform2 = TransformContainer(transform=second_valid_transform)
+        transform_program2.push_back(transform2)
+
+        transform_program = transform_program1 + transform_program2
+
+        assert len(transform_program) == 2
+
+        assert isinstance(transform_program[0], TransformContainer)
+        assert transform_program[0].transform is first_valid_transform
+
+        assert isinstance(transform_program[1], TransformContainer)
+        assert transform_program[1].transform is second_valid_transform
+
+        transform_program = transform_program2 + transform_program1
+
+        assert len(transform_program) == 2
+
+        assert isinstance(transform_program[0], TransformContainer)
+        assert transform_program[0].transform is second_valid_transform
+
+        assert isinstance(transform_program[1], TransformContainer)
+        assert transform_program[1].transform is first_valid_transform
```

Comment:
```suggestion
```
Duplicate code

Reply:
No it is different because it is tranform2 + transform1